ontology/yaml/resources/states/states.yaml

```diff
@@ -23,6 +23,10 @@ MEDIUM: Medium speed or output setting.
 HIGH: High speed or output setting.
 OCCUPIED: Occupied sensor state or operation mode.
 UNOCCUPIED: Unoccupied sensor state or operation mode.
+VFD: Discrete speed (frequency) control
+AUTOLOW: Low speed auto mode
+AUTOMEDIUM: Medium speed auto mode
+AUTOHIGH: High Speed auto mode
```

Comment:
Remove these; high, medium, and low are already defined. Use those.

Reply:
This has been explained as above. 
<html>
<body>
<!--StartFragment--><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px !important; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">The following shows the available mode:</p>

Raw | Mode
-- | --
0 | OFF
1 | HIGH
2 | MEDIUM
3 | LOW
4 | AUTOHIGH
5 | AUTOMEDIUM
6 | AUTOLOW
7 | VFD

<!--EndFragment-->
</body>
</html>

Reply:
the AUTO* has been removed from states.yaml